### PR TITLE
connect bookmark more button to context menu

### PR DESCRIPTION
**Pull request by SeanConn15**
Signed-off-by: Sean Connelly seanconn256@protonmail.com

### CODE REVIEW

**Comment by SeanConn15** [comment]
This is my first contribution, so there may be something I missed here. Thanks to anyone who helps me here.

**Comment by biodranik** [comment]
Thanks! Looks like the code can be simplified, and the already existing code may be reused, @Endem10n what do you think?
Java sometimes sounds too verbose for simple, already implemented tasks.

File: android/src/com/mapswithme/maps/bookmarks/BookmarkCategoriesAdapter.java [hunk 1]
```diff
@@ -230,6 +239,23 @@ public void onShowAll()
       notifyDataSetChanged();
     }
   }
```
Comment (Endem10n) [inline android/src/com/mapswithme/maps/bookmarks/BookmarkCategoriesAdapter.java:241]: @SeanConn15 could you, please, add empty line between methods' bodies?

Comment (Endem10n) [inline android/src/com/mapswithme/maps/bookmarks/BookmarkCategoriesAdapter.java:241]: @SeanConn15 could you, please, add empty line between methods' bodies?

**Comment by Endem10n** [comment]
@biodranik LGTM. In this case all code that added in PR is needed. Not sure, that we should unite some of listeners for bookmark list. Need to think it over a bit in background, if we need to refactor some code in bookmarks functionality in different places and how exactly. For now I suggest to merge PR. I'd prefer to do some refactoring in bookmarks later in separate PR.

**Review by rtsisyk — COMMENTED** [review]
The patch follows existing style and solves #824 problem, as described. I agree that original code has too many useless abstraction levels. However, this is not a problem of this patch. 

Approved. Thanks for the contribution!

**Review by rtsisyk — APPROVED** [review]

**Comment by SeanConn15** [comment]
added space and signed off.

**Review by biodranik — APPROVED** [review]

**Pull request merged**